src/operator/subgraph/subgraph_property.h

```diff
@@ -221,6 +222,11 @@ class SubgraphProperty {
     return nullptr;
   }
 
+  virtual void PrePartition(const nnvm::Graph& g,
```

Comment:
Can you provide an example how to use this API? Please take `excluded_ops=['BatchNorm']` as an example.

Reply:
Got your point. Because in this PR, Prepartition and PostPartition are just 2 empty virtual function, so I can't image the usage of them. Your explanation makes sense to me. But I still suggest to implement  `excluded_ops` to `default` backend as an example(considering `default` backend is an example, we can use it to example the features newly added), then other developer can know how to use this 2 APIs.